external-crates/move/move-command-line-common/src/character_sets.rs

```diff
@@ -33,21 +35,32 @@ pub fn is_permitted_newline_crlf_chars(c1: char, c2: char) -> bool {
     is_cr && is_lf
 }
 
+// Taken from https://en.wikipedia.org/wiki/Bidirectional_text
+// TODO Double check 200F 200E 061C
+const UNICODE_TEXT_DIRECTIONALITY_CONTROL_CHARS: &[char] = &[
+    '\u{202A}', '\u{202B}', '\u{202C}', '\u{202D}', '\u{202E}', '\u{2066}', '\u{2067}', '\u{2068}',
+    '\u{2069}',
+];
+
+/// Is some other non-whitespace unicode character
+pub fn is_permitted_unicode(c: char) -> bool {
```

Comment:
It's a permitted unicode if it's not a ascii and not a whitespace is difficult to relate to for me.
Maybe just the function name or I am not really following...